babybear/.gitignore

```diff
@@ -0,0 +1,162 @@
+# This gitignore has been specially created by the WPILib team.
```

Comment:
Asher ... you can't checkin in this directory.  Sorry.  Put it in the proper directory please should look like: camp/abolotski/*

Reply:
This is not a mini-project, it's the code for the new pre-season bot.

Reply:
ok